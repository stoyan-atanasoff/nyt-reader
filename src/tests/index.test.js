describe('Entry point', () => {
  it('should render and match the snapshot', () => {
    document.body.innerHTML = '<div id="root"></div>';
    const index = require('../index');
    expect(index).toMatchSnapshot();
  });
});

