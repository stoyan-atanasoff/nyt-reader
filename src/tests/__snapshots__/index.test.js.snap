// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Entry point should render and match the snapshot 1`] = `Object {}`;
